BlueprintUI/Sources/BlueprintView/BlueprintView.swift

```diff
@@ -40,6 +40,8 @@ public final class BlueprintView: UIView {
 
     private var sizesThatFit: [SizeConstraint: CGSize] = [:]
 
+    private var cacheStorage = Environment.CacheStorageEnvironmentKey.defaultValue
```

Comment:
I think we need to do some sort of garbage collection on this, right? Eg, evict paths that are no longer present after a layout update?